RustLeanModels/Basic.lean

```diff
@@ -13,19 +12,23 @@ abbrev Str := List Char
 
 --LOGIC
 
-lemma exists_minima_prop_nat {p: Nat → Prop} : (∃ x , p x) → ∃ m, (p m) ∧ (∀ n, p n → m ≤ n) :=by
+theorem exists_minima_prop_nat {p: Nat → Prop} : (∃ x , p x) → ∃ m, (p m) ∧ (∀ n, p n → m ≤ n) :=by
   intro gn; obtain ⟨n, gn⟩ := gn
```

Comment:
Small simplification: you can do it in one step.
```
   intro ⟨n, gn⟩
```

Reply:
Indeed!